Zend/tests/traits/direct_static_member_access.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+Cannot directly access static members on a trait
+--FILE--
+<?php
+
+trait T {
+    public static $foo;
+    public static function foo() {
+        echo "Foo\n";
+    }
+}
+
+class C {
+    use T;
+}
+
+try {
+    T::$foo = 42;
+    var_dump(T::$foo);
```

Comment:
These 2 lines should be in their own try/catch block, otherwise the case covered by the 2nd line would not be tested.